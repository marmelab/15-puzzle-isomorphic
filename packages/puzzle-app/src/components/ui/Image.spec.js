import React from 'react';
import renderer from 'react-test-renderer';

import Image from './Image';

describe('Image', () => {
    test('should render Image without error', () => {
        renderer.create(<Image src="path/to/image" />);
    });

    test('should render the Image correctly', () => {
        const component = renderer.create(<Image src="path/to/image" />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
