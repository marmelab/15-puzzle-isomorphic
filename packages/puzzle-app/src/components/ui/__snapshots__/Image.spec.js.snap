// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render the Image correctly 1`] = `
<div
  className="image-wrapper"
>
  <img
    alt={undefined}
    className="responsive-img"
    src="path/to/image"
  />
</div>
`;
